_components/form-templates.md

```diff
@@ -175,61 +178,12 @@ lead: Patterns for some of the most commonly used forms on government websites.
   </div>
 </div>
 
-<div class="preview">
```

Comment:
@carodew for some reason I'm not seeing a duplicate SSN form on staging. I'm seeing just one.